libr/asm/p/asm_bf.c

```diff
@@ -58,80 +58,64 @@ static int disassemble(RAsm *a, RAsmOp *op, const ut8 *buf, int len) {
 	return rep;
 }
 
+static bool _write_asm(RAsmOp *op, int value, int n) {
+	ut8 *opbuf = malloc (n);
```

Comment:
use calloc and remove the memset at line 66

Reply:
What should I use instead of memset here? (Just to be clear - my intention was to fill the buffer with copies of `value`)

Reply:
oh wait. i'm stupid. sorry. i read that malloc + memset as malloc + bzero behaviour...
Ignore this. i'll merge it as is.

Reply:
Thanks!